src/components/StaticContent/FAQs.re

```diff
@@ -75,9 +75,9 @@ module FAQs = {
                   )}>
                   <div>
                     <div className=faqExplainerText>
-                      <Text>
+                      <Text className=Styles.colorGreen>
                         <S>
-                          "Yes! but not all of them. Some represent real animals and some like Simon and Andy don't but all the funds raised by the animals go to the conservations."
+                          "Some wildcards do represent real animals, while other wildcards are symoblic of the coservation efforts of that organisation. When partnering with a conservation organisation, if that organisation has specific ainimals they track and care for, we aim to list those real animals. In other cases, conservation efforts can occur on a more macro level (without specific animals). To raise funds for these efforts we create a fictional animal representative of that organisation. In both cases, an animal will simply represent an organisation, and funds generated from that animal will flow to the organisation it represents. "
```

Comment:
Cool, this is great, until we start utilising our "disaster relief/social security animals"... And our OGs are also wanderers.

Reply:
I don't love using the word "fictional",  maybe we say "digital animal". Not too sure. Dont mind if you want to keep it as is.

Reply:
Noted and changed.